### ISSUE TITLE
Suggestion/Permission

### ISSUE THREAD

**Issue by Misunderstood-Wookiee** [kaytotes/ImprovedBlizzardUI#116]
I sent you a msg on Curse unsure if you visit curse, essentially I queried and expressed interest in maintaining my own version of this on curse, and was hoping we may be able to come to an agreement. 

Obviously, I do not intend to claim what is not my code or work. I also intended too fully kickback to this project so I thought maybe I could maintain my own version on curse which atm it is loaded in my addons and git as ImprovedBlizzardUIPlus, until such time as you decide to take it back over and I can always take down my version when it suits you. 

I was just not sure if or when would want to make time to merge a PR or update curse and maybe it might be easier for both of us if I had a version going that people could use in meantime. I understand if no, or if you wanted to collab possibly have two flavours going on the original curse project that could be doable. 

Let me know thoughts, as I am not sure if you would mind but I do need clarification via a response otherwise cures will reject my project as it is similar to yours and your licence does not allow derived works. Just to clarify I am not going to steal your work, I simply derived upon it and if you look over my git for the project you will see I mention you in as many ways as applicable and this project.

**Comment by Misunderstood-Wookiee** [comment]
No worries, I will maybe be sleep by then due to timezone differences, I operate under AU EST+10 
trying to track down-a nill colour bug atm which rarely appears when finishing a pet battle have only see it once so far unsure of

 another addon such as Pet Tracker or Collect Me is trying to write or change the same frame data or if the tooltip.lua is missing a catch to apply data. 

What I can see is this [Tooltop Lua Error.txt](https://github.com/kaytotes/ImprovedBlizzardUI/files/2438627/Tooltop.Lua.Error.txt)

and near the bottom, it is trying to set a nil colour for the pet frame I believe.

**Comment by Misunderstood-Wookiee** [comment]
No worries, the change log everything mostly I keep it pretty tidy. Mainly
just adjusted a few fonts and defaults to fit better into frames when the
names exceed 14chrs. Split the font up into two fonts so it is possible too
separate the ui from the chat style.

Added thye ability too hide everything in the main bar gfx wise. Guild
default colour is calm lime ish green to bettet match bliz default.

Fixed a major bug and I'm looking into implementing shared media lib.

So fonts and textures can be pulled from a source where quite a few othe
addons use it for a global media dir..

WeakAuras and details damage meter are a couple who use shared media lib
when it's installed to extended the font and texture assets of the inbuilt.

But understand you reason for saying no that's okay. I'm just trying to
learn how make addons and I wanted too do some stuff with the ui and I been
using this one for a while actually since I came back from leaving in cats
I choose this as I hate the truly minimal ones I like blizzard style.

So decided I'd make adjustments to this and see how it goes.
